scripts/check_for_changed_paymentsheet_screenshots.rb

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
+
+require 'open3'
+
+diff_command = 'git status --porcelain paymentsheet-example/screenshots/debug'
+stdout, _, _ = Open3.capture3(diff_command)
+
+if stdout.empty?
+    puts "No differences in screenshots found."
+    exit true
+end
+
+puts "Screenshot tests failed.\n\n#{stdout}\n\nScreenshots are in the artifacts tab in bitrise."
```

Comment:
Can we link to this?

Reply:
Good idea. Looks like we have access to environment variables for the build url, then we can just append a query param for the tab.

I'll do this in a follow up.

Thanks!